LogMonitor/src/LogMonitor/LogWriter.h

```diff
@@ -13,14 +13,13 @@ class LogWriter final
         InitializeSRWLock(&m_stdoutLock);
 
         DWORD dwMode;
-        hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
 
-        if (!GetConsoleMode(hConsole, &dwMode)) {
+        if (!GetConsoleMode(GetStdHandle(STD_OUTPUT_HANDLE), &dwMode))
+        {
             m_isConsole = false;
         }
-        else {
-            m_isConsole = true;
-        }
+
+        m_isConsole = true;

```

Comment:
This is the default value. Should be before the `if` else we'll be resetting line 19

Reply:
reverted this change to what was working before, need to investigate the impact of this change further, addressing as new PR
```c++
void FlushStdOut()
    {
        if (m_isConsole)
        {
            fflush(stdout);
        }
    }
```